tests/package/packagers_testers/python_standard_library_packagers_testers.py

```diff
@@ -47,17 +49,12 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 
-NoneType = type(None)  # TODO: Replace with types.NoneType from python 3.10
-
-
-def pack_none() -> NoneType:
+def pack_none() -> None:
```

Comment:
type hinting `-> None` means it is not returning anything as `None` is a actually an instance of `NoneType`. So the hint is correct (see the TODO above about Python 3.10)

Reply:
I don't think so. We can discuss it, but please note that not returning anything from a function is equivalent to `return None`. It is indeed a special case where the type hint is not the class of the returned object.

Reply:
well, I guess both are correct. I wen't on the pure logic approach I guess. If you think its better this way, let it be :)